tests/wsgi/tests/test_util.py

```diff
@@ -72,7 +71,10 @@ def test_build_precache_map():
 def test_event():
     event = {
         "database": "enwiki",
-        "meta": {"topic": "mediawiki.revision-create"},
+        "meta": {
+            "topic": "eqiad.mediawiki.revision-create",
```

Comment:
There will be no 'topic' in the event at all, you can delete this from the test.